state/Fetchable.js

```diff
@@ -6,6 +6,14 @@ import { refreshToken } from './token.js';
  */
 export class FetchError extends Error {}
 
+export class FetchServerError extends FetchError {
+	constructor(status, message) {
+		super(status);
```

Comment:
```suggestion
		super(message);
```

Since `FetchError` is basically [a marker interface for `Error`](https://github.com/BrightspaceHypermediaComponents/foundation-engine/blob/776f844a04834085c3e94646df6d875d0c83801f/state/Fetchable.js#L7), which [has `message` as its first parameter](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Error/Error). Could then drop `errorMessage` in favour of the standard `message` property.

Reply:
Thanks, sounds good.

Reply:
Oh actually I like that status is the call to super since it's not changing existing behavior and message may be empty.

Reply:
Yeah, not changing existing behaviour might just be the best thing (even if I think the original way is odd).

Reply:
Yeah, not changing existing behaviour might just be the best thing.